PaolaSantolloV/react-apprenticeship-capstone4
# Feat/deliverable1

## Description
This PR contain the first deliverable

- Add home page
- Add layout UI

## Type of Change


## How should this be tested?

In your terminal run `npm install` after that` npm run start,` now you can see the page on[ localhost:3000](https://localhost:3000)
To run the test `npm run test`, if you want to watch the coverage` npm run test a -- --coverage`.

## What is the deliverable?

Deliverable 1

## Screenshots and links
netlify link: [ https://wonderful-nightingale-bce5f1.netlify.app/]( https://wonderful-nightingale-bce5f1.netlify.app/)


## Checklist
- [x ] I added tests to prove that my fix is effective or my deliverable works.
